components/hubspot/sources/deal-updated/deal-updated.mjs

```diff
@@ -5,9 +5,33 @@ export default {
   key: "hubspot-deal-updated",
   name: "Deal Updated",
   description: "Emit new event each time a deal is updated.",
-  version: "0.0.12",
+  version: "0.0.13",
   type: "source",
   dedupe: "unique",
+  props: {
+    ...common.props,
+    pipeline: {
+      propDefinition: [
+        common.props.hubspot,
+        "dealPipeline",
+      ],
+      description: "Filter deals by pipeline",
+      optional: true,
+    },
+    stage: {
+      propDefinition: [
+        common.props.hubspot,
+        "stages",
+        (c) => ({
+          pipeline: c.pipeline,
+        }),
+      ],
+      type: "string",
+      label: "Stage",
+      description: "Filter deals by stage",
+      optional: true,
+    },
+  },
   hooks: {},
```

Comment:
do we need an empty hooks property?

Reply:
This is to overwrite the hooks section in `common.mjs`.